### ISSUE TITLE
youtube wayback machine capture fallback

### ISSUE THREAD

**Issue by MrRawes** [yt-dlp/yt-dlp#3342]
### Checklist

- [X] I'm reporting a feature request
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2022.03.08.1** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Description

this feature request is a option for if the youtube video that the user tries to download is unavailable it will fallback to a wayback machine capture of the video

**Comment by pukkandan** [comment]
Use `ytarchive:video_id`

**Comment by pukkandan** [comment]
Yes, what they are asking is to support some pseudo URL pattern like `ytf:id` to try downloading id in both YouTube and archive. The other request is impossible because it was suggesting a single way to support archive pages of all websites. But this is only about YouTube, for which we already support web archive.
